src/components/title-search-filters.js

```diff
@@ -1,92 +1,52 @@
-  { label: 'Streaming Audio', value: 'streamingaudio' },
-  { label: 'Streaming Video', value: 'streamingvideo' },
-  { label: 'Thesis & Dissertation', value: 'thesisdissertation' },
-  { label: 'Website', value: 'website' },
-  { label: 'Unspecified', value: 'unspecified' }
-];
-
-export default function TitleSearchFilters({
-  filter = {},
-  onUpdate
-}) {
-  let { type = 'all', selected } = filter;
 
+import SearchFilters from './search-form/search-filters';
+
+/**
+ * Renders search filters with specific title filters.
+ *
+ * Once the API supports returning these filters via an endpoint, this
+ * component should not be necessary. Instead the `search-form` should
+ * recieve the available filters from the route and render the
+ * search-filters component itself.
+ */
+export default function TitleSearchFilters(props) {
   return (
-    <div data-test-eholdings-title-search-filters>
-      <Accordion
-        name="selected"
-        label="Selection Status"
-        separator={false}
-        header={FilterAccordionHeader}
-        displayClearButton={!!selected}
-        onClearFilter={() => onUpdate({ ...filter, selected: undefined })}
-        closedByDefault={false}
-      >
-        {selectedFilters.map(({ label, value }, i) => (
-          <RadioButton
-            key={i}
-            name="selected"
-            id={`eholdings-title-search-filters-selected-${value}`}
-            label={label}
-            value={value}
-            checked={value === selected}
-            onChange={() => onUpdate({ ...filter, selected: value })}
-          />
-        ))}
-      </Accordion>
-
-      <hr />
-
-      <Accordion
-        name="pubtype"
-        label="Publication Type"
-        separator={false}
-        header={FilterAccordionHeader}
-        displayClearButton={type !== 'all'}
-        onClearFilter={() => onUpdate({ ...filter, type: undefined })}
-        closedByDefault={false}
-      >
-        {pubtypeFilters.map(({ label, value }, i) => (
-          <RadioButton
-            key={i}
-            name="pubtype"
-            id={`eholdings-title-search-filters-pubtype-${value}`}
-            label={label}
-            value={value}
-            checked={value === type}
-            onChange={() => onUpdate({ ...filter, type: value })}
-          />
-        ))}
-      </Accordion>
-    </div>
+    <SearchFilters
+      searchType="titles"
+      availableFilters={[{
+        name: 'selected',
+        label: 'Selection Status',
+        defaultValue: 'all',
+        options: [
+          { label: 'All', value: 'all' },
+          { label: 'Selected', value: 'true' },
+          { label: 'Not Selected', value: 'false' },
+          { label: 'Selected By EBSCO', value: 'ebsco' }
+        ]
+      }, {
+        name: 'type',
+        label: 'Publication Type',
+        defaultValue: 'all',
+        options: [
+          { label: 'All', value: 'all' },
+          { label: 'Audio Book', value: 'audiobook' },
```

Comment:
Since we're in here, sentence case 😎

Reply:
Totally forgot. Thanks!

Reply:
Are we sentence casing the section headings `"Selection status"` and `"Publication type"` too? Or leave these title-cased?